examples/prompting/Basic_Classification.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    a) Topic: What can I do after highschool?
</pre><p>If you update the prompt, can you remove the a) b) c) prefixes too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>